lib/manager.js

```diff
@@ -166,9 +81,15 @@ Manager.prototype.pollRunningInstances = function(){
     if (instance.lastCheckedBalance - self.calculateCharge(token) <= self._minBalance) {
       self.chargeToken(token).then(function(newBalance){
         if (newBalance < self._minBalance) {
-          instance.runner.kill();
-          winston.debug('Killed token "' + token + '"');
-          delete instance;
+          request.del('http://127.0.0.1:'+((parseInt(process.env.PORT) || 5000) + 30)+
```

Comment:
In the future perhaps this behavior could be extracted into some kind of `ComputeServiceClient` library.